src/runtime-rs/config/configuration-dragonball.toml.in

```diff
@@ -207,8 +207,6 @@ container_pipe_size=@PIPESIZE@
 internetworking_model="@DEFNETWORKMODEL_DB@"
 
 name="@RUNTIMENAME@"
-hypervisor_name="@HYPERVISOR_DB@"
-agent_name="@PROJECT_TYPE@"
```

Comment:
Could you rebase and remove these options from the other runtime-rs config files?